### PR TITLE
add module website_remove_powered_by_odoo

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: website_remove_powered_by_odoo/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Website Remove Powered by Odoo
+==============================
+
+* This module extends the functionality of website and allows you to remove Powered by Odoo from the website.
+* Note: This module interits the template from the webstie_sale. It may has conflict with certain themes.
```
Comment (elicoidal) [inline website_remove_powered_by_odoo/README.rst]: s/interits/inherits
s/webstie/website
s/has/have

Comment (elicoidal) [inline website_remove_powered_by_odoo/README.rst]: This note should be in issue/roadmap section

File: website_remove_powered_by_odoo/website_remove_powered_by_odoo.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <template id="metallion_layout" inherit_id="website.layout">
```
Comment (elicoidal) [inline website_remove_powered_by_odoo/website_remove_powered_by_odoo.xml]: id should fixed

Comment (Rona111) [inline website_remove_powered_by_odoo/website_remove_powered_by_odoo.xml]: @elicoidal   fix lilke "<template id="layout" inherit_id="website.layout">"??

**Comment by elicoidal** [comment]
LGTM

**Pull request merged**